docs-kits/kits/PCF Exchange Kit/page_adoption-view.md

```diff
@@ -505,12 +496,9 @@ The diagram shown here illustrates the interaction between the PCF KIT and the o
 
 ## STANDARDS
 
-The standards for release 24.03 will be published soon. Our relevant standards can be downloaded from the official [Catena-X Standard Library](https://catena-x.net/de/standard-library):
+The standards this release will be published soon. Our relevant standards can be downloaded from the official [Catena-X Standard Library](https://catena-x.net/de/standard-library):
```

Comment:
There seems to be something off with the first sentence. Perhaps "The standard **of** this ..."